openconcept/thermal/tests/test_heat_pipe.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
 from openmdao.utils.assert_utils import assert_near_equal, assert_check_partials, assert_warning
 from openmdao.api import Problem, NewtonSolver, DirectSolver
-import openconcept.components.heat_pipe as hp
+import openconcept.thermal.heat_pipe as hp
```

Comment:
Is heat pipe a comp?  Why doesn't it follow the camel case naming scheme of other components?

Reply:
It's a file, just changed the import scheme so it matches others